gapic/generator/options.py

```diff
@@ -57,26 +59,28 @@ def build(cls, opt_string: str) -> 'Options':
             # Set the option.
             # Just assume everything is a list at this point, and the
             # final instantiation step can de-list-ify where appropriate.
-            opts.setdefault(opt, [])
-            opts[opt].append(value)
+            opts.setdefault(opt[13:], [])
+            opts[opt[13:]].append(value)
```

Comment:
I'm struggling to understand why it's `opt[13:]`. A comment or a named constant would've made sense here.

Reply:
`len('python-gapic-')` is equal to 13.
I will change it to that.